Esri/arcgis-maps-sdk-swift-samples
# [New] Select feature in scene layer

## Description

This PR implements **`Select feature in scene layer `** in `Scenes` category.

`swift/issues/5433`

## How To Test

Tap a feature in the SceneView and check to see that it is highlighted. Select another feature and ensure the previous selection is removed when the new selection is highlighted.

## Screenshots

![select-features-in-scene-layer](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/13937686/01c320c0-8c72-4bf2-8568-fdc5b44631ad)
